lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -48,8 +51,19 @@ export class NativeInteractionClient extends BaseInteractionClient {
 
         const reqTimestamp = TimeUtils.nowSeconds();
         const response: object = await this.nativeMessageHandler.sendMessage(messageBody);
-        const validatedResponse: NativeResponse = this.validateNativeResponse(response);
-        return this.handleNativeResponse(validatedResponse, nativeRequest, reqTimestamp);
+        const validatedResponse: NativeResponse = this.validateNativeResponse(response, nativeATMeasurement);
+
+        // generate the authentication result
+        const authResult = await this.handleNativeResponse(validatedResponse, nativeRequest, reqTimestamp, nativeATMeasurement);
+
+        // end the perf measurement
+        nativeATMeasurement.endMeasurement({
+            success: true,
+            isNativeBroker: true
+        });
+        nativeATMeasurement.flushMeasurement();
```

Comment:
Don't need to call `flushMeasurement` here, only needs to be called at the top level API.

Reply:
What is `top level API`? Should we not flush once we no longer need the measurement? I may be mixing up the definitions, can you please point to the docs?

Reply:
A top level API would be `acquireTokenSilent`, for example (i.e. an API invoked by teh application). Internal measurements should just call start and stop, not flush.

Reply:
`flushMeasurement` will emit the measurements to the application, which should not be done until the top level API is about to complete.

Reply:
Got it. I used flush the same way, I wanted to clean it up when NativeMeasurement ends. 

So here will be the flow:
silent flow measurement starts -> native flow measurement starts -> native flow measurement is flushed (emitted to the application) -> silent flow measurement ends (in silentSSO)

Reply:
Clarified `flush` functionality, the `clearance` is based on `correlationId` and hence cannot flush early. Added to the top level now.